includes/templates/admin/user/date-time.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * User Date Time Settings Template.
+ *
+ * @package GatherPress\Core
+ * @since 0.29.0
+ */
+
+use GatherPress\Core\Utility;
+
+if ( ! isset( $timezone ) && ! isset( $date_format ) && ! isset( $time_format ) ) {
+	return;
+}
+
+$gatherpress_tz_choices = Utility::timezone_choices();
+
+$gatherpress_date_attrs = array(
+	'name'  => 'gp_date_format',
+	'value' => ! empty( $date_format ) ? $date_format : '',
+);
+
+$gatherpress_time_attrs = array(
+	'name'  => 'gp_time_format',
+	'value' => ! empty( $time_format ) ? $time_format : '',
+);
+?>
+<div style="margin-top: 40px; margin-bottom: 30px;">
+	<h2 id="gp-user-date-time">
+		<?php esc_html_e( 'Date & Time Formatting', 'gatherpress' ); ?>
+	</h2>
+	<div>
+		<?php esc_html_e( 'For more information read the <a href="https://wordpress.org/documentation/article/customize-date-and-time-format/">Documentation on date and time formatting</a>.', 'gatherpress' ); ?>
```

Comment:
This is going to escape the markup. Maybe echo with __ wrapped in KSES function.